test/rules.jl

```diff
@@ -158,26 +158,43 @@ _second(t) = Base.tuple_type_head(Base.tuple_type_tail(t))
             ChainRulesCore.frule(::Any, ::typeof(first_oa), x::T, y::T) where {T<:Float32}
         )
 
+        # forgotten ChainRulesCore prefix #545
```

Comment:
I suggest a bit longer to be a bit clearer.
Also prefer to use full URLs rather than issue numbers for easy clickability.
```suggestion
        # Declared without using the ChainRulesCore namespace qualification
        # see https://github.com/JuliaDiff/ChainRulesCore.jl/issues/545
```